app/scripts/modules/core/src/presentation/main.less

```diff
@@ -974,6 +974,18 @@ select:invalid {
   }
 }
 
+.filter-search {
+  input::placeholder {
+    font-size: 14px;
+    color: var(--color-titanium);
+  }
+  .glyphicon {
```

Comment:
This makes me a little uncomfortable, especially now that we're moving towards SVG icons (sort of like the same queasiness I get when I see new code in AngularJS).  Is this styling making the "?" render nicely?  What does the default styles look like?

Reply:
Noted. I will find more specific non-glyphicon related classnames. This comes from the`HelpField` component, which is defaulted as a small, dark grey, question mark. I can utilize all of the functionality of that component, but need it to visually be bigger and lighter.

Reply:
gotcha.  

I bet this use of glyphicon in HelpField will eventually be caught in the great-svg-icon-refactor of 2020 :)